tune/tune.c

```diff
@@ -81,9 +81,9 @@ static int exfat_get_volume_label(struct exfat_blk_dev *bd, off_t root_clu_off)
 		return -1;
 	}
 
-	memset(volume_label, 0, 11);
+	memset(volume_label, 0, VOLUME_LABEL_MAX_LEN);
```

Comment:
Correct way how to erase whole statically allocated buffer is `memset(buffer, 0, sizeof(buffer))`.